doc/install/plugin.rst

```diff
@@ -9,29 +9,29 @@ Linux install
 ~~~~~~~~~~~~~
 
 Obtain the Linux plugin from the link above. These instructions assume that you
-have installed Trelis in ``/opt/Trelis-16.0``. If you installed it somewhere
+have installed Cubit in ``/opt/Coreform-Cubit-2020.2``. If you installed it somewhere
 else, modify these instructions appropriately.
 
 These operations may need to be performed as the root user (sudo).
 
-1.  Unpack the tarball in ``/opt/Trelis-16.0/bin/plugins``.
-2.  Change directory to ``/opt/Trelis-16.0/bin/plugins/dagmc``.
+1.  Unpack the tarball in ``/opt/Coreform-Cubit-2020.2/bin/plugins``.
+2.  Change directory to ``/opt/Coreform-Cubit-2020.2/bin/plugins/dagmc``.
 3.  Run the install script: ``./install.sh``.
 
 OS/X install
 ~~~~~~~~~~~~
 
 Obtain the OS/X plugin from the link above. These instructions assume that you
-have installed Trelis in ``/Applications/Trelis-16.0``. If you installed it
+have installed Cubit in ``/Applications/Coreform-Cubit-2020.2.app``. If you installed it
 somewhere else, modify these instructions appropriately.
 
 These operations may need to be performed as the root user (sudo).
 
 1.  If it does not already exist, create a folder called ``plugins`` in
-    ``/Applications/Trelis-16.0/Contents/MacOS``.
-2.  Unpack the tarball in ``/Applications/Trelis-16.0/Contents/MacOS/plugins``.
+    ``/Applications/Coreform-Cubit-2020.2.app/Contents/MacOS``.
+2.  Unpack the tarball in ``/Applications/Coreform-Cubit-2020.2.app/Contents/MacOS/plugins``.
 3.  Change directory to
-    ``/Applications/Trelis-16.0/Contents/MacOS/plugins/dagmc``.
+    ``/Applications/Coreform-Cubit-2020.2.app/Contents/MacOS/plugins/dagmc``.
```

Comment:
Are these paths also correct, then, @bam241 (I just updated them based on the above fix)?